bowler/main.py

```diff
@@ -138,5 +138,24 @@ def run(codemod: str, argv: List[str]) -> None:
         sys.argv[1:] = original_argv
 
 
+@main.command()
+@click.argument("codemod", required=True, type=str)
+def test(codemod: str) -> None:
+    """
+    Run the tests in the codemod file 
+        """
+    import unittest
+    import importlib.util
+    import os.path
+
+    module_name_from_codemod = os.path.basename(codemod).replace(".py", "")
```

Comment:
I would expect this import code to be pretty similar to the 'run' case above, can you check?